packages/api/src/external/carequality/patient.ts

```diff
@@ -88,14 +88,13 @@ async function prepareForPatientDiscovery(
 ): Promise<OutboundPatientDiscoveryReq> {
   const { cxId } = patient;
   const fhirPatient = toFHIR(patient);
-  const nearbyOrgs = await searchCQDirectoriesAroundPatientAddresses({ patient });
-  const nearbyWithUrls = nearbyOrgs.filter(org => org.urlXCPD);
+  const nearbyOrgsWithUrls = await searchCQDirectoriesAroundPatientAddresses({ patient });
   const orgOrderMap = new Map<string, number>();
-  nearbyWithUrls.forEach((org, index) => {
+  nearbyOrgsWithUrls.forEach((org, index) => {
     orgOrderMap.set(org.id, index);
   });
```

Comment:
OOC, why do we overwrite the org's ID w/ a sequential integer?

Reply:
We don't overwrite the IDs. Since these orgs are already ordered by their distance from the patient address, storing them in a map like this is good for faster sorting of the other gateways that we get in the process

Reply:
I was probably almost sleeping when I wrote that. 😅